wodles/aws/buckets_s3/aws_bucket.py

```diff
@@ -669,6 +665,9 @@ def check_bucket(self):
             elif error_code == INVALID_REQUEST_TIME_ERROR_CODE:
                 error_message = INVALID_REQUEST_TIME_ERROR_MESSAGE
                 exit_number = 19
+            else:
+                error_message = f"Unexpected error: {error}"
```

Comment:
Please follow the same error guideline being used in the integration.

- Declare the error message at the beginning of the file.
Ex:

```python
UNKNOWN_ERROR_MESSAGE = "Unexpected error: '{error}'."
```
- Format the error for it to be logged.

Ex:
```suggestion
                error_message = UNKNOWN_ERROR_MESSAGE.format(error=error)
```

- Do the same for the `server_access` file error handling (import the error).

Reply:
After these changes:

```
# /var/ossec/wodles/aws/aws-s3 --bucket test-bucket --aws_profile default --aws_account_id 123456789123 --regions us-west-2 --type cloudtrail --debug 2
DEBUG: +++ Debug mode on - Level: 2
DEBUG: Generating default configuration for retries: mode standard - max_attempts 10
ERROR: Unexpected error: 'An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist'.
```